### PR TITLE
Project Initialization

### PR DESCRIPTION
**Pull request by eaesmat**
Use React Routers 
Use React Components

### CODE REVIEW

File: src/Components/Circular.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import './Circular.css';
+
+const Circle = () => '';
+
+export default Circle;
```
Comment (elmar8287) [inline src/Components/Circular.js]: - [ ] I think this component does not use in your project. Please, consider removing it, to not be confused 👍

File: src/Components/Books.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          <ul className="list-inline d-flex pt-2">
+            <li className="text-primary pe-2">Comment</li>
+            <div className="vr" />
+            <li className="text-primary px-2 ">Remove</li>
+            <div className="vr" />
+            <li className="text-primary ps-2 ">Edit</li>
+          </ul>
+        </div>
+        <div className="col-3" />
+        <div className="col-3 border-start">
```
Comment (elmar8287) [inline src/Components/Books.js]: - [ ] Please, consider the requirements of this milestone:

- `Renders a list of books using the Book component`

It means that your Books component should be the Book component. 

The Book component should be, as it requered:

- Takes the book data as a prop and renders the title and the author
- Has a button for removing books (the button wouldn't remove the books *yet*, this is just a placeholder for now)

So, as a result, you have to create a Book component separately and use it inside of Books one.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @eaesmat.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- You have made some styling - good job 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- Please, try to make informative PR descriptions in the future - your PR will be more professional 👍 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by medaminedev66 — APPROVED** [review]
Hi @eaesmat 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- Highly organized/clean code.
- The repo looks professional.
- The web app runs without errors.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**